cmd/serve/serve.go

```diff
@@ -48,11 +48,10 @@ var ServeCmd = &cobra.Command{
 			password := os.Getenv("K8SGPT_PASSWORD")
 			model := os.Getenv("K8SGPT_MODEL")
 			baseURL := os.Getenv("K8SGPT_BASEURL")
-
-			// If the envs are set, allocate in place to the aiProvider
+			engine := os.Getenv("K8SGPT_ENGINE")
+			// If the envs are set, alocate in place to the aiProvider
 			// else exit with error
-			envIsSet := backend != "" || password != "" || model != "" || baseURL != ""
-
+			envIsSet := backend != "" || password != "" || model != "" || baseURL != "" || engine != ""
```

Comment:
Engine should not be mandatory for serve under different backends

Reply:
I think same goes for baseurl.
OpenAI's backend  doesn't require a base url to work.
WDYT?

Reply:
Agreed